cockatrice/src/carddatabase.cpp

```diff
@@ -336,7 +336,7 @@ QString PictureLoader::getPicUrl()
     else
         picUrl = picDownloadHq ? settingsCache->getPicUrlHqFallback() : settingsCache->getPicUrlFallback();
 
-    picUrl.replace("!name!", QUrl::toPercentEncoding(card->getCorrectedName()));
+    picUrl.replace("!name!", QUrl::toPercentEncoding(card->getCorrectedName().toLower()));
     picUrl.replace("!cardid!", QUrl::toPercentEncoding(QString::number(muid)));
```

Comment:
maybe we could add a new tag named like !lowcasename! instead of replacing the current one?